components/train.yaml

```diff
@@ -106,6 +108,22 @@ implementation:
           ${bootleg_args} \
           $@
 
+      if test "$calibrate" == "true" ; then
+        genienlp predict \
+            --path "$modeldir" \
+            --data "dataset" \
+            --eval_dir "eval_dir" \
+            --task ${task_name} \
+            --silent \
+            --overwrite \
+            --evaluate valid \
+            --save_confidence_features \
+            --confidence_feature_path "$modeldir/confidence_feature_file.pkl" \
+            ${calibration_additional_args}
```

Comment:
"calibration_additional_args" should be passed to `calibrate` right? If you need arguments to predict, you need another parameter.

Reply:
This is the arguments fed to `genienlp predict` for the purpose of calibration.